keyboards/bioi/c11/rules.mk

```diff
@@ -0,0 +1,34 @@
+MCU = atmega32u4
+F_CPU = 8000000
+ARCH = AVR8
+F_USB = $(F_CPU)
+BOOTLOADER = qmk-dfu
+BOOTLOADER_SIZE = 4096
+
+
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+
+SRC += 	../usart.c \
+        ../ble.c \
+        ../main.c
+
+OPT_DEFS += -DPROTOCOL_BLE
+OPT_DEFS += -DUART_RX1_BUFFER_SIZE=16 -DUART_TX1_BUFFER_SIZE=16
+OPT_DEFS += -DUSART1_ENABLED
+
+# Build Options
+BOOTMAGIC_ENABLE = no
+MOUSEKEY_ENABLE = no
+EXTRAKEY_ENABLE = yes
+CONSOLE_ENABLE = no
+COMMAND_ENABLE = yes
+NKRO_ENABLE = no
+BACKLIGHT_ENABLE = no
+AUDIO_ENABLE = no
+UNICODE_ENABLE = no
+BLUETOOTH_ENABLE = no
+RGB_MATRIX_ENABLE = yes
+RGB_MATRIX_DRIVER = WS2812
+SLEEP_LED_ENABLE = no
+LTO_ENABLE = yes
+VIA_ENABLE = yes
```

Comment:
All of this should be moved to the info.json. 

Also, via should not be enabled at the kb level, and needs to be enabled only at the keymap level.